cppcache/integration-test/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_library(test-cppcache-utils STATIC
 target_link_libraries(test-cppcache-utils
   PRIVATE
     ACE::ACE
+    Boost::boost
```

Comment:
Do you need all of Boost? You should be able to include just the libraries you need, Boost::asio, for example.

Reply:
After changing CacheStatistics this is not needed anymore. Thanks!